### ISSUE TITLE
Incorrect result from multiplication.

### ISSUE THREAD

**Issue by ZCG-coder** [ZCG-coder/Steppable#5]
## Describe the bug

When invoking the `multiply` function, or using the `*` operator in Python, an incorrect result may occur.

## To Reproduce

1. Create two `steppyble` numbers in the form of `A0...B`, e.g.: `203`, `2003`...
2. Invoke the `*` operator to multiply the both.
3. The result is incorrect. For example:
   ```pycon
   >>> steppyble.Number("2003") * steppyble.Number("2003")
   4606900
   >>> 2003 * 2003
   4012009
   ```

## OS

- **ALL PLATFORMS**

**Comment by ZCG-coder** [comment]
It turns out that `power` is affected by this too.

**Comment by ZCG-coder** [comment]
Fixed in [30d1cde](https://github.com/ZCG-coder/Steppable/commit/30d1cde958c3038ca8160b691a7312f5703fc420).
